lang/csharp/build.sh

```diff
@@ -26,6 +26,9 @@ VERSION=`cat $ROOT/share/VERSION.txt`
 case "$1" in
 
   test)
+    # AVRO-2442: Explictly set LANG to work around ICU bug in `dotnet test`
+    LANG=en_US.UTF-8
```

Comment:
Should we explicitly export this in case the shell doesn't already have it marked for export? Or alternatively, would it make sense to put `LANG=en_US.UTF-8` just at the start of line 34 so it doesn't affect anything other than the test run itself?

Reply:
@kojiromike, good call. I'll move it down to the beginning of line 34. Nothing else needs this little workaround at this time.